lang.json

```diff
@@ -20,4 +20,11 @@
 	"view-page": "Go to the thai page",
 	"url": "th/",
 	"index": "th/index.html"
+},{
+	"lang": "ID",
```

Comment:
Can you write for "lang" and "title" attribute your language (here "Indonesian") in your local language (Indonesian) please?